src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -5854,6 +5864,27 @@ namespace CondenserLoopTowers {
             }
             if (this->WaterTemp > (TempSetPoint + state.dataCondenserLoopTowers->towers(this->VSTower).MaxRangeTemp)) { // run flat out
                 OutletWaterTempLocal = this->WaterTemp - state.dataCondenserLoopTowers->towers(this->VSTower).MaxRangeTemp;
+            } else {
+                if (this->WaterTemp > TempSetPoint) {
```

Comment:
My head is exploding here trying to figure out what `TempSetPoint < this->WaterTemp < TempSetPoint + MaxRangeTemp` means (i.e., where we are right now inside this if).